lib/network/modules/View.js

```diff
@@ -154,10 +153,10 @@ class View {
 
   /**
    *
-   * @param {Object} options  |  options.offset   = {x:Number, y:Number}   // offset from the center in DOM pixels
-   *                          |  options.time     = Number                 // animation time in milliseconds
-   *                          |  options.scale    = Number                 // scale to animate to
-   *                          |  options.position = {x:Number, y:Number}   // position to animate to
+   * @param {Object} options  |  options.offset   = {x:number, y:number}   // offset from the center in DOM pixels
```

Comment:
Again, not from you but a more valid construct here would be:

```
 * @param {Object} options
 * @param options.offset {x:number, y:number} - bla bla
```
Actually I'm not sure if that's correct either....but it's better than what's there now.

This comment applies to futher occurrences as well.